js/plates-view/render-cross-section.ts

```diff
@@ -337,8 +337,8 @@ class CrossSectionRenderer {
       return;
     }
     // Based on the dropped-pin.png size.
-    const kImgWidth = 21;
-    const kImgHeight = 26;
+    const kImgWidth = 48 * 0.42;
+    const kImgHeight = 72 * 0.42;
```

Comment:
Perhaps a comment or constant explaining the `0.42`?

Reply:
Assets are kinda in a draft state. This one was missing, so I had to modify another one by hand, and adjust the size. I guess it'll be soon replaced by the final version.

Reply:
Just pushed a comment, thanks.